serverless.yml

```diff
@@ -124,6 +127,19 @@ stepFunctions:
                 - States.ALL
                 Next: errorHandler
             Next: fieldVisitTransform
+          tsFieldVisitReadings:
+            Type: Task
+            Resource: arn:aws:lambda:${self:provider.region}:${self:custom.accountNumber}:function:aqts-capture-field-visit-metadata${self:provider.stage}-preProcess
+            Parameters:
+              id.$: $.id
+              partitionNumber.$: $.partitionNumber
+            ResultPath: $.tsFieldVisitResult
+            TimeoutSeconds: 105
+            Catch:
+              - ErrorEquals:
+                - States.ALL
+                Next: errorHandler
+            Next: fieldVisitTransform
```

Comment:
Rather than `fieldVisitTransform` as the next state to trigger in this etl, let's use the `defaultSuccess` state instead.  The `fieldVisitTransform` state won't know what to do with the shredded metadata from the tables you've created yet.  In a future task, we can handle this.